MessengerOnFileForLinux/Common/test/TimeManagmentTest.cpp

```diff
@@ -0,0 +1,42 @@
+#include <gtest/gtest.h>
+#include <ctime>
+#include <regex>
+#include <iostream>
+
+#include "TimeManagment.hpp"
+
+TEST(ConvertTimeToString, TimeFormatSizeIsNineteen)
+{
+    const auto timeAsString = TimeManagment::convertTimeToString(TimeManagment::getActualDate());
```

Comment:
misuse auto line 10-11

Reply:
What is wrong with auto here ?

Reply:
as upper, do not use auto everywhere, constexpr int is better.